code/game/objects/items/weapons/teleprod.dm

```diff
@@ -23,6 +23,9 @@
 
 /obj/item/weapon/melee/baton/cattleprod/attackby(obj/item/I, mob/user, params)//handles sticking a crystal onto a stunprod to make a teleprod
 	..()
+	if(istype(src, /obj/item/weapon/melee/baton/cattleprod/sparkprod))
```

Comment:
This is a terrible place to put this.  The message will show if you use anything but a cell on a sparkprod.